src/Service.Tests/SqlTests/GraphQLMutationTests/MultipleCreateMutationTests/MultipleCreateMutationTestBase.cs

```diff
@@ -0,0 +1,819 @@
+        /// In this request, the second Book item in the input violates the create policy defined. Processing of that input item is expected to result in database policy violation error.
+        /// All the items created successfully prior to this fault input will also be rolled back. So, the end result is that no new items should be created. 
+        /// </summary>
+        public async Task ManyTypeMultipleCreateFailsDueToCreatePolicyFailure(string expectedErrorMessage, string bookDbQuery, string publisherDbQuery)
+        {
+            string graphQLMutationName = "createbooks";
+            string graphQLMutation = @"mutation {
+                                          createbooks(
+                                            items: [
+                                              { title: ""Book #1"", publisher_id: 2345 }
+                                              { title: ""Test"", publisher_id: 2345 }
+                                            ]
+                                          ) {
+                                            items {
+                                              id
+                                              title
+                                              publishers {
+                                                id
+                                                name
+                                              }
+                                            }
+                                          }
+                                        }";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true, clientRoleHeader: "role_multiple_create_policy_tester");
+
+            SqlTestHelper.TestForErrorInGraphQLResponse(
+                response: actual.ToString(),
+                message: expectedErrorMessage,
+                statusCode: $"{DataApiBuilderException.SubStatusCodes.DatabasePolicyFailure}");
+
+            // Validate that no book item is created
+            string dbResponse = await GetDatabaseResultAsync(bookDbQuery);
+            Assert.AreEqual("[]", dbResponse);
+
+            // Validate that no publisher item is created
+            dbResponse = await GetDatabaseResultAsync(publisherDbQuery);
+            Assert.AreEqual("[]", dbResponse);
+        }
+
+        /// <summary>
+        /// Point type multiple create mutation request is executed with the role: role_multiple_create_policy_tester. This role has the following create policy defined on "Reviews" entity: "@item.websiteuser_id ne 1".
+        /// In this request, the second Review item in the input violates the read policy defined. Hence, it is not to be returned in the response.
+        /// The returned response is validated against an expected response for correctness.
```

Comment:
```suggestion
        /// Point type multiple create mutation request is executed with the role: role_multiple_create_policy_tester. 
        /// This role has the following create policy defined on "Reviews" entity: "@item.websiteuser_id ne 1".
        /// In this request, the second Review item in the input violates the read policy defined. Hence, it is not to be returned in the response.
        /// The returned response is validated against an expected response for correctness.
```

> This role has the following create policy defined on "Reviews" entity: "@item.websiteuser_id ne 1".
Isn't this test validating behavior of read policy violation? the mutation inserts a review with websituser_id == 1 which violates the create policy. 
The follow up read shouldn't occur at all because of mutation error.

Reply:
description needs to be updated since expected response is demonstrating that read policy restricts values in selection set.

Reply:
`@item.websiteuser_id ne 1` is acutally a read policy. Fixed the comment. 

Updated the description to call out that read policies are honored when constructing the response.